docs/articles/documentation/test-api/assertions/assertion-api.md

```diff
@@ -48,6 +48,16 @@ await t
     .expect({ a: 'bar' }).eql({ a: 'foo' }, 'this assertion will fail');
 ```
 
+```js
+import { Selector } from 'testcafe';
```

Comment:
Do we forget specify `page`?

Reply:
I think we can leave it as is. Since it's not a real page but just an example hot to use api we can skip the page directive.

Reply:
I don't mind